src/pluginHooks.js

```diff
@@ -1451,6 +1451,16 @@ const REGISTERED_HOOKS = [
    */
   'beforeUndo',
 
+  /**
+   * Fired by {@link UndoRedo} plugin before changing undo stack.
+   *
+   * @event Hooks#beforeUndoStackChange
```

Comment:
Can I ask you to add a `@since` tag for each newly added hook?

Reply:
Done: https://github.com/handsontable/handsontable/pull/7827/commits/9d7c94ed445e321eb168c8293ac30540808c89de.